src/utils/ddb/ddb_commands.c

```diff
@@ -1451,3 +1451,65 @@ ddb_run_dtx_stat(struct ddb_ctx *ctx, struct dtx_stat_options *opt)
 
 	return rc;
 }
+
+int
+ddb_run_dtx_aggr(struct ddb_ctx *ctx, struct dtx_aggr_options *opt)
+{
+	struct dv_indexed_tree_path itp = {0};
+	daos_handle_t               coh = {0};
+	uint64_t                   *epoch;
+	int                         rc;
+
+	if (!ctx->dc_write_mode) {
+		ddb_error(ctx, error_msg_write_mode_only);
+		return -DER_INVAL;
+	}
+
+	rc = init_path(ctx, opt->path, &itp);
+	if (!SUCCESS(rc))
+		goto done;
+
+	if (!itp_has_cont(&itp)) {
+		rc = -DER_INVAL;
+		goto done;
+	}
+
+	rc = dv_cont_open(ctx->dc_poh, itp_cont(&itp), &coh);
+	if (!SUCCESS(rc))
+		goto done;
+
+	switch (opt->format) {
+	case DDB_DTX_AGGR_NOW:
+		epoch = NULL;
+		break;
+	case DDB_DTX_AGGR_DATE:
+		rc = ddb_date2epoch(optarg, &opt->epoch);
+		if (!SUCCESS(rc)) {
+			ddb_error(ctx, "'--date' option arg date format is invalid\n");
+			goto done;
+		}
+	case DDB_DTX_AGGR_EPOCH:
+		epoch = &opt->epoch;
+		break;
+	default:
+		D_ASSERT(false && "Invalid dtx_aggr options");
+	}
+
+	ddb_print(ctx, "Starting DTX entries aggregation of ");
+	itp_print_full(ctx, &itp);
+	ddb_print(ctx, "\n");
+	rc = vos_dtx_aggregate(coh, epoch);
```

Comment:
Each time, `vos_dtx_aggregate()` will remove at most one committed DTX entries blob, that maybe not enough for the whole DTX aggregation process. We may have to repeatedly call `vos_dtx_aggregate()` until the expected aggregated entries up to the given condition.

Reply:
> Oups...
> 
> * [x]  Fix all DTX entries blob

Fixed with commit c2b4bb7ed